pkgs/development/libraries/blitz/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, fetchurl, pkgconfig, gfortran, texinfo, python
+
+# Select SIMD alignment width (in bytes) for vectorization.
+, simdWidth ? 1
+
+# Pad arrays to simdWidth by default?
+# Note: Only useful if simdWidth > 1
+, enablePadding ? false
+
+# Activate serialization through Boost.Serialize?
+, enableSerialization ? true, boost ? null
+
+# Activate test-suite?
+# WARNING: Some of the tests require up to 1700MB of memory to compile.
+, doCheck ? true
+
+}:
+
+assert enableSerialization -> boost != null;
+
+let
+  inherit (stdenv.lib) optional optionals;
+in
+
+stdenv.mkDerivation rec {
+  name = "blitz++-1.0.1";
+  src = fetchurl {
+    url = "https://github.com/blitzpp/blitz/archive/1.0.1.tar.gz";
+    sha256 = "1vyw795jxg7d9ac7hlybink0jgkrybjcbzh1gcq69ck4ggqc6bxn";
+  };
+
+  nativeBuildInputs = [ pkgconfig python ];
+  buildInputs = [ gfortran texinfo ]
```

Comment:
I think texinfo also belongs to `nativeBuildInputs`.

Reply:
I will take your word for it.